Sources/ScipioKit/Producer/PIF/PIFGenerator.swift

```diff
@@ -263,6 +274,31 @@ private struct PIFLibraryTargetModifier {
         settings[.SWIFT_INSTALL_OBJC_HEADER] = "YES"
 
         configuration.buildSettings = settings
+
+        var impartedBuildProperties = configuration.impartedBuildProperties
+        var imparted = impartedBuildProperties.buildSettings
+        if var flags = imparted[.OTHER_CFLAGS] {
+            flags.removeAll { $0.hasPrefix("-fmodule-map-file") }
+            imparted[.OTHER_CFLAGS] = flags
+        }
```

Comment:
You can avoid mutatable vars.

```suggestion
        // Remove all `-fmodule-map-file` settings
        imparted[.OTHER_CFLAGS] = imparted[.OTHER_CFLAGS]?.filter { !$0.hasPrefix("-fmodule-map-file") }
```

Reply:
`filter` で削除するのって、残すものを書くのでややこしいと思っているんですが、
`var` もややこしいので、せっかくなのでsuggestionもらいます。

Reply:
気持ちはわかる